config/default.yml

```diff
@@ -1080,6 +1080,11 @@ Layout/SpaceAroundKeyword:
   Enabled: true
   VersionAdded: '0.49'
 
+Layout/SpaceAroundMethodCallOperator:
+  Description: 'Checks method call operators to not have spaces around them.'
+  Enabled: true
```

Comment:
New cops should be "pending" by default.

Reply:
Changed to pending & updated docs.